services/delivery/src/deliver/get-ad.js

```diff
@@ -39,7 +37,15 @@ const updateCorrelator = async (correlator, ad, lineitemId) => {
  */
 module.exports = async (correlator, adunit, date) => {
   const correlated = await db.findOne('correlators', { value: correlator });
-  if (correlated) return correlated;
+  if (correlated) {
+    // Return the correlated item's ad src/url
+    const ad = await db.strictFindActiveById('ads', correlated.adId, { projection: { 'image.src': 1, url: 1 } });
+    return {
+      ...correlated,
+      src: ad.image.src,
```

Comment:
In this case `src` -- `serveSrc` is a virtual on the mongoose `image` model, but it is no longer used. It was replaced by the cdnHostname from the publisher (aka serve.inddist.com), which the delivery app accounts for within the `/image/` route

Reply:
Oops, actually you're right -- this was previously reading from the correlator, which was build from the schedule, which was writing it from the `srvSrc`: https://github.com/parameter1/email-x/blob/77aa695b210883100f0ff5017d747a2aa6148317/services/graphql/src/mongoose/schema/lineitem.js#L245

For any publisher without a custom CNAME, this would be the env variable for the serve domain (aka example.serve.email-x.parameter1.com)...with this change it is now the raw S3 website URL hostname. Will get this updated too.